images/airflow/2.10.1/python/mwaa/config/airflow.py

```diff
@@ -318,4 +331,5 @@ def get_opinionated_airflow_config() -> Dict[str, str]:
         **_get_opinionated_airflow_core_config(),
         **_get_opinionated_airflow_scheduler_config(),
         **_get_opinionated_airflow_secrets_config(),
+        **_get_usage_data_config(),
```

Comment:
nit: All other methods are following the convention of starting with `_get_opinionated_`. Maybe follow the same here.

Reply:
Fixed the naming to be consistent